src/components/aktivitetskrav/vurdering/VurderAktivitetskravSkjemaButtons.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Button } from "@navikt/ds-react";
+import { ButtonRow } from "@/components/Layout";
+
+const texts = {
+  lagre: "Lagre",
+  avbryt: "Avbryt",
+};
+
+interface VurderAktivitetskravSkjemaButtonsProps {
+  isSubmitting: boolean;
+  handleClose: () => void;
+}
+
+export const VurderAktivitetskravSkjemaButtons = ({
+  isSubmitting,
+  handleClose,
+}: VurderAktivitetskravSkjemaButtonsProps) => {
+  return (
+    <ButtonRow>
+      <Button loading={isSubmitting} type="submit">
+        {texts.lagre}
+      </Button>
+      <Button variant="tertiary" onClick={handleClose}>
+        {texts.avbryt}
+      </Button>
+    </ButtonRow>
```

Comment:
Nå har ikke jeg jobbet veldig mye med react, men jeg tror jeg foretrekker å bare ha det i samme fil når det er så enkelt som 2 knapper. Jeg mister litt oversikten der denne blir brukt, og må trykke inn på komponenten for å skjønne hva `VurderAktivitetskravSkjemaButtons` er. Der det i stedet bare kunne vært en submit og avbryt knapp i klartekst der det ble brukt.
Trenger ikke å endre noe, bare tanker jeg har om slik oppdeling 😊

Reply:
Skjønner hva du mener her, men føler det treffer godt på hva definisjonen av en gjenbrukbar komponent i react kan være 🤔 Hadde et navnebytte gjort det mer leselig, Vetle?
Feks `LagreOgAvbrytKnapperad` eller noe sånt?

Reply:
Ja, uheldig valg av navn her - gjenbrukbarheten har jo egentlig ikke noe med aktivitetskrav å gjøre. LagreOgAvbrytKnapperad er mye bedre og mer på linje med hvordan man vil dele opp i gjenbrukbare komponentene, tenker jeg.